modelcard.md

```diff
@@ -18,20 +18,26 @@ metrics:
 model-index:
 - name: {model_id}
   results:
-  - task: 
-      type: {task_type}  # Required. Example: automatic-speech-recognition
-      name: {task_name}  # Optional. Example: Speech Recognition
+  - task:
+      type: {task_type}             # Required. Example: automatic-speech-recognition
+      name: {task_name}             # Optional. Example: Speech Recognition
+      verified: {evaluation_type}   # Optional. Indicates if evaluation self-reported or generated by Hugging Face. Example: false
```

Comment:
I've added this new field to distinguish evaluation results that are generated by users vs those that will come from our evaluation service. Internal Slack thread: https://huggingface.slack.com/archives/C035S5G2J3D/p1650448897906539?thread_ts=1650438388.214969&cid=C035S5G2J3D